src/components/organisms/WorldEntranceForm/WorldEntranceForm.tsx

```diff
@@ -37,13 +38,18 @@ export interface WorldEntranceFormProps {
   world: WithId<World>;
 }
 
+let codeQuestionsCopy: Question[] | undefined = [];
+let profileQuestionsCopy: Question[] | undefined = [];
+let entranceCopy: EntranceStepConfig[] | undefined = [];
```

Comment:
`const` instead of `let`.
no shared mutable state (which is why there is `ALWAYS_EMPTY_ARRAY`)

Reply:
I guess, the question would be, should this be `const a = ALWAYS_EMPTY_ARRAY` or should be `const [a,setA]=useState()`

Reply:
got rid of those `let`s 941192c